src/App/App.js

```diff
@@ -4,10 +4,9 @@ import {
   Route, 
   // Redirect
 } from 'react-router-dom';
-// import { Link } from 'react-router-dom';
 import {LoginForm} from '../LoginForm/LoginForm'
 import { Homepage } from '../Homepage/Homepage'
-import { ErrorBoundary } from '../ErrorMessage/ErrorMessage.js';
+import { ErrorBoundary }  from '../ErrorBoundary/ErrorBoundary';
```

Comment:
I'm glad changing file names wasn't "as much" of as hassle as it was the first time.